src/libraries/Native/Unix/System.Security.Cryptography.Native/android_security.c

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Out of curiosity, why is this file *C* and not C++?

Reply:
Whole .NET libraries PAL layer is in C for size and portability reasons